src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/RegionAnalysisTests.vb

```diff
@@ -6628,6 +6857,8 @@ End Module
             Assert.Empty(dataFlowResults.CapturedOutside)
             Assert.Equal("x", GetSymbolNamesJoined(dataFlowResults.DataFlowsIn))
             Assert.Empty((dataFlowResults.DataFlowsOut))
+            Assert.Equal("x", GetSymbolNamesJoined(dataFlowResults.DefinitelyAssignedOnEntry))
+            Assert.Equal(Nothing, GetSymbolNamesJoined(dataFlowResults.DefinitelyAssignedOnExit))
```

Comment:
surprisin result.  for operators, are we missing LeaveRegion?  i would expect 'x' to be def assigned here.

Reply:
@gafter @agocke